rework usdc tutorial (#1950)
smartcontractkit/documentation: src/content/ccip/tutorials/usdc.mdx

## Example

In this tutorial, you will learn how to send USDC tokens from a smart contract on Avalanche Fuji to a smart contract on Ethereum Sepolia using Chainlink CCIP and pay CCIP fees in LINK tokens.

> The process will involve the following steps:

Reviewer: ```suggestion
The process uses the following steps:
```

>       1.  **Transfer USDC and Data:** You'll initiate a transfer of USDC tokens and associated data from the Sender contract on Avalanche Fuji. The data will include the function signature of the stake function of the Staker contract along with the necessary arguments.

Reviewer: ```suggestion
      1.  **Transfer USDC and Data:** Initiate a transfer of USDC tokens and associated data from the Sender contract on Avalanche Fuji. The data includes the required arguments and the signature of the `stake` function from the Staker contract.
```

>       1.  **Receive and Stake:** The Receiver contract on Ethereum Sepolia will receive the tokens and data. It will then directly use this data to make a low-level call to the Staker contract, executing the stake function to stake the USDC on behalf of a beneficiary.

Reviewer: ```suggestion
      1.  **Receive and Stake:** The Receiver contract on Ethereum Sepolia receives the tokens and data. Then, it uses this data to make a low-level call to the Staker contract, executing the `stake` function to stake USDC on behalf of a beneficiary.
```

>       1.  **Redeem Staked Tokens:** The beneficiary will later be able to redeem the staked tokens for USDC.

Reviewer: ```suggestion
      1.  **Redeem Staked Tokens:** The beneficiary can redeem the staked tokens for USDC later.
```

> The purpose of including the function signature and arguments in the data is to demonstrate how arbitrary data can support a variety of scenarios and use cases. By sending specific instructions within the data, you can define various interactions between smart contracts across different blockchain networks, making your decentralized application more flexible and powerful.

Reviewer: ```suggestion
The purpose of including the function signature and arguments in the data is to demonstrate how arbitrary data can support a variety of scenarios and use cases. By sending specific instructions within the data, you can define various interactions between smart contracts across different blockchain networks and make your decentralized application more flexible and powerful.
```

<ClickToZoom src="/images/ccip/tutorials/usdc-tutorial.jpg" alt="Chainlink CCIP usdc tutorial" />

      <Aside type="note">
      After you followed the tutorial:

         - You can read the [Explanation](#explanation) section to understand the smart
         contracts' logic and how they interact with CCIP.

>          - Explore the [CCIP USDC
>          masterclass](https://cll-devrel.gitbook.io/ccip-masterclass-4) to learn about other examples of cross-chain transfers with
>          USDC.

Reviewer: ```suggestion
         - Explore the [CCIP USDC masterclass](https://cll-devrel.gitbook.io/ccip-masterclass-4) to find other examples of cross-chain transfers with USDC.
```
